src/gol.js

```diff
@@ -0,0 +1,37 @@
+export default function next(input) {
+  const output = input.map((row) => row.map((cell) => cell));
+  /*
+    input.map(function(row) {
+      return row.map(function(cell) {
+        return cell
+      })
+    })
+  */
+
+  for (let i = 0; i < input.length; i += 1) {
+    for (let j = 0; j < input[i].length; j += 1) {
+      const liveNeighbours = countLiveNeighbours(input, i, j);
+      if (liveNeighbours == 3) {
+        output[i][j] = 1;
+      } else if (liveNeighbours < 2  || liveNeighbours > 3) {
+        output[i][j] = 0;
+      } else {
+        output[i][j] = input[i][j];
+      }
+    }
+  }
+  return output;
+}
+
+export function countLiveNeighbours(board, i, j) {
```

Comment:
It is a good practice to define functions before you use them so this function should be placed before next function